mPowerSDKTests/APHScoreCalculatorTests.m

```diff
@@ -2,8 +2,33 @@
 //  APHScoreCalculatorTests.m
 //  mPowerSDK
 //
-//  Created by Shannon Young on 1/5/16.
-//  Copyright © 2016 Sage Bionetworks. All rights reserved.
+// Copyright (c) 2015, Sage Bionetworks. All rights reserved.
```

Comment:
2016

fine to leave the "Created by..." line up above, if you like. Or not. Whatever you prefer.